### PR TITLE
List and navbar

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
please provide review

### CODE REVIEW

File: component/list/list.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.drawer-list{
+    display: inline-block;
+    grid-area: d;
+    color: var(--black);
+    background-color: var(--white);
+    overflow: auto;
+    border-radius: 5px;
+}
+.drawer-unordered-list{
```
Comment (anshulraheja) [inline component/list/list.css:8]: In this class, you could have added display flex property. 
The result would be that
1. You would not have to define drawer-list-item as inline-block (line no 16)
2. You could have reused this unordered class in the stack list too and there you would only have to change the flex-direction to the column.

File: component/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.navbar-component{
+    display: flex;
+    background-color: var(--black);
```
Comment (anshulraheja) [inline component/navbar/navbar.css:9]: Just a suggestion, use a different background color for the navbar as its difficult to see it

**Review by anshulraheja — APPROVED** [review]

File: component/list/list.css (hunk 1 above)
Comment (dev-ashishsethi) [reply component/list/list.css:8]: Thanks for suggestion Anshul! I will fix it

File: component/navbar/navbar.css (hunk 2 above)
Comment (dev-ashishsethi) [reply component/navbar/navbar.css:9]: fixed it. thanks

**Comment by SinchanaMS** [comment]
A couple of suggestions, if you want to consider:
1. cursor: pointer on the list items on hover would be better. 
2. Equal spacing of the items in spaced lists would look better.

File: component.css [hunk 3]
```diff
@@ -18,6 +21,9 @@
 h1,h2,h3,h4,h5,h6{
     color: var(--white);
```
Comment (rahulyadav139) [inline component.css]: instead of setting colours for each (text) element, you can set font colour in the body tag directly.

**Review by rahulyadav139 — APPROVED** [review]

File: component/navbar/navbar.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
+.navbar-container{
+    background-color: var(--white);
+    border-radius: 8px;
+    padding: 3rem ;
+}
+
+.navbar-component{
+    display: flex;
+    background-color: var(--black);
```
Comment (manojsarna) [inline component/navbar/navbar.css:9]: color selection for the backgrounds can be improved.

File: component/navbar/navbar.css [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    margin-bottom: 3px;
+    margin-left:1rem;
+}
+.navbar-title{
+    margin-right: auto;   
+    color: var(--red);
+    font-size: 32px;
+    font-weight: 600;
+    margin-left: 1rem;
+}
```
Comment (manojsarna) [inline component/navbar/navbar.css:28]: use rem for font size..!!

**Review by manojsarna — APPROVED** [review]
overall looks good.

can be merged.!!

**Review by dhruvin25799 — APPROVED** [review]
Looks good, clean UI!
Minor suggestion: Add cursor pointer for list items and transition to hover items for the list, makes it look a bit smoother.

**Pull request merged**